src/lib/types.ts

```diff
@@ -77,3 +77,17 @@ export interface FacebookAccessToken {
 }
 
 // #endregion
+
+// #region --- About Us Page ---
+export interface Organisation {
+  name: string;
+  url: string;
+}
```

Comment:
tu jeszcze bedzie logo i nazwy w ogole nie wyswietlamy, możemy jej za to użyć jako alt
```suggestion

// #region --- About Us Page ---
export interface Organisation {
  name: string;
  url: string;
  logo: string;
}
```

Reply:
Jak tu na przyklad
<img width="1112" alt="Zrzut ekranu 2024-12-18 o 22 16 45" src="https://github.com/user-attachments/assets/31965b18-b10b-4f93-b794-6dc00de2e8a0" />